src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1247,6 +1248,24 @@ public Jail getPrimaryJail() {
 		return primaryJail;
 	}
 	
+	// Returns the number of players imprisoned in the town as an integer
+	public int getJailedPlayerCount(List<Resident> residentsJailed) {
+		int JailedPlayerCount = residentsJailed.size();
+		return JailedPlayerCount;
+	}
+
+	// Iterates through residents in town and adds to Resident List if jailed
+	public List<Resident> getJailed() {
+
+		List<Resident> residentsJailed = new ArrayList<>();
+
+		for (Resident resident : getResidents()) {
```

Comment:
This method will not count outlaws or prisoners of war in the town jail system.

Reply:
Method overhauled to pull from TownyUniverse getJailedResidentMap and sort a new resident list based on those with matching jail town.